content/en/blog/_posts/2023-11-16-mid-cycle-1.29.md

```diff
@@ -0,0 +1,77 @@
+
+For more information about how to enable and run external cloud controller managers, read [Cloud Controller Manager Administration](/docs/tasks/administer-cluster/running-cloud-controller/) and [Migrate Replicated Control Plane To Use Cloud Controller Manager](/docs/tasks/administer-cluster/controller-manager-leader-migration/).
+
+For general information about cloud controller managers, please see
+[Cloud Controller Manager](/docs/concepts/architecture/cloud-controller/) in the Kubernetes documentation.
+
+For general information about cloud controller managers, please see the Kubernetes document
+[Cloud Controller Manager](/docs/concepts/architecture/cloud-controller/).
+
+### Removal of `v1beta2` flow control API group
+
+The _flowcontrol.apiserver.k8s.io/v1beta2_ API version of FlowSchema and PriorityLevelConfiguration will [no longer be served](/docs/reference/using-api/deprecation-guide/#v1-29) in Kubernetes v1.29. 
+
+To prepare for this, you can edit your existing manifests and rewrite client software to use the `flowcontrol.apiserver.k8s.io/v1beta3` API version, available since v1.26. All existing persisted objects are accessible via the new API. Notable changes in `flowcontrol.apiserver.k8s.io/v1beta3` include
+that the PriorityLevelConfiguration `spec.limited.assuredConcurrencyShares` field was renamed to `spec.limited.nominalConcurrencyShares`.
+ 
+
+### Deprecation of `status.nodeInfo.kubeProxyVersion` field for Node
+
+The `.status.kubeProxyVersion` field for Node objects will be [marked as deprecated](https://github.com/kubernetes/enhancements/issues/4004) in v1.29 in preparation for its removal in a future release. This field is not accurate and is set by kubelet, which does not actually know the kube-proxy version, or even if kube-proxy is running.
+    
+## Major Changes for Kubernetes v1.29
+
+### Native sidecar containers promoted to beta
+
+Introduced in alpha in Kubernetes v1.28, native sidecare containers will be [beta](https://github.com/kubernetes/enhancements/issues/753) in Kubernetes v1.29. In addition to fixing known issues, improvements include implementing proper termination ordering and allowing sidecar containers to restart during the shutdown of the Pod.
+
+### Want to know more?
+
+Deprecations are announced in the Kubernetes release notes. You can see the announcements of pending deprecations in the release notes for:
+
+* [Kubernetes v1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md#deprecation)
+* [Kubernetes v1.26](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.26.md#deprecation)
+* [Kubernetes v1.27](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md#deprecation)
+* [Kubernetes v1.28](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.28.md#deprecation)
```

Comment:
This seems wrong (see feedback originally posted to https://hackmd.io/@krol/middlecycle-129), but we could go with it.